packages/mds-db/tests/mds-db.spec.ts

```diff
@@ -14,30 +14,32 @@
  * limitations under the License.
  */
 
-import assert from 'assert'
 /* eslint-reason extends object.prototype */
-/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
+/* eslint-disable prettier/prettier */
+/* eslint-disable @typescript-eslint/no-unused-vars */
 import should from 'should'
```

Comment:
I'm surprised this import belongs between `eslint-enable` and `eslint-disable`

Reply:
If you don't disable, eslint will remove the unused import and the build will fail. I discovered.